docs/sources/open-source.md

```diff
@@ -8,42 +8,64 @@ title: Open Source
+Grafana OnCall is a developer-friendly incident response tool that's available to Grafana open source and Grafana Cloud users. The OSS version of Grafana OnCall provides the same reliable on-call management solution along with the flexibility of a self-managed environment. 
+
+This guide describes the necessary installation and configuration steps needed to configure OSS Grafana OnCall. 
+
+The intended audience for this guide includes:
+- Grafana open source admins who are responsible for deploying and configuring Grafana OnCall.
+- Grafana open source users who need to configure SMS and phone notifications using Grafana Cloud.
+
+## Install Grafana OnCall OSS
+
+There are three Grafana OnCall OSS environments available:
+
+- **Hobby** playground environment for local usage: [README.md](https://github.com/grafana/oncall#getting-started)
 - **Development** environment for contributors: [DEVELOPER.md](https://github.com/grafana/oncall/blob/dev/DEVELOPER.md)
-- **Production** environment for reliable cloud installation using Helm: [Production Environment](#production-environment)
+- **Production** environment for reliable cloud installation: [Production Environment](#production-environment)
+
+For detailed installation instructions and additional resources, refer to the OSS Grafana OnCall [README.md](https://github.com/grafana/oncall#getting-started)
+
+For more information on production environment installation, refer to the following OSS Grafana OnCall [production environment helm chart](https://github.com/grafana/oncall/helm)
+
+
+## Slack integration for Grafana OnCall OSS
 
-## Production Environment
+The Slack integration for Grafana OnCall leverages Slack API features to provide a customizable and useful integration.
 
-We prepared the helm chart for production environment: https://github.com/grafana/oncall/helm
+### Before you begin
 
-## Slack Setup
+Review the following requirements to configure Slack for Grafana OnCall OSS:
 
-Grafana OnCall Slack integration use a lot of Slack API features: 
-- Subscription on Slack events requires OnCall to be externally available and provide https endpoint. 
-- You will need to register new Slack App.
+- Ensure your Grafana OnCall environment is running.
+- Grafana OnCall must be externally available and provide HTTPS endpoint to establish subscription on Slack events.
+- Create a new Slack App from Grafana OnCall app manifest.
 
-1. Make sure your OnCall is up and running.
+### Configure Slack for Grafana OnCall OSS
+
+1. Ensure your Grafana OnCall environment is up and running.
+
+2. Grafana OnCall must be accessible through HTTPS. For development purposes, it's recommend to use [localtunnel](https://github.com/localtunnel/localtunnel). For production purposes, consider establishing a proper web server with HTTPS termination. 
+For localtunnel, refer to the following configuration:
```

Comment:
```suggestion
1. Ensure your Grafana OnCall environment is up and running.

1. Grafana OnCall must be accessible through HTTPS. For development purposes,  use [localtunnel](https://github.com/localtunnel/localtunnel). For production purposes, consider establishing a proper web server with HTTPS termination. 
For localtunnel, refer to the following configuration:
```

Everything under this step needs to be indented by two tab spaces, beginning with "For localtunnel,..."